fastly/provider.go

```diff
@@ -28,6 +28,7 @@ func Provider() terraform.ResourceProvider {
 		},
 		ResourcesMap: map[string]*schema.Resource{
 			"fastly_service_v1":                         resourceServiceV1(),
+			"fastly_service_wasm":                       resourceServiceWasmV1(),
```

Comment:
We are waiting on confirmation of this naming.

Reply:
Naming has been confirmed and updated